plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -0,0 +1,1466 @@
+    type: str
+    choices: [ merged, deleted ]
+    default: merged
+  config:
+    description:
+    - List of details of Authentication and Policy Servers being managed.
+    type: list
+    elements: dict
+    required: true
+    suboptions:
+      authentication_policy_server:
+        description: Manages the Authentication and Policy Servers.
+        type: dict
+        suboptions:
+          server_type:
+            description:
+            - Type of the Authentication and Policy Server.
+            - ISE for Cisco ISE servers.
+            - AAA for Non-Cisco ISE servers.
+            type: str
+            choices: [ AAA, ISE ]
+            default: AAA
+          server_ip_address:
+            description: Ip Address of the Authentication and Policy Server.
+            type: str
+            required: True
+          shared_secret:
+            description:
+            - Shared secret between devices and authentication and policy server.
+            - Shared secret key length should be from 4 to 10.
+            - Shared secret is a Read-Only parameter.
+            type: str
+          protocol:
+            description:
+            - Type of protocol for authentication and policy server.
+            - RADIUS provides centralized services (AAA) for users in remote access scenarios.
+            - TACACS focuses on access control and administrative authentication for network devices.
+            type: str
+            choices: [ TACACS, RADIUS, RADIUS_TACACS]
+            default: TACACS
+          encryption_scheme:
+            description:
+            - Type of encryption scheme for additional security.
+            - Updation of encryption scheme is not possible.
+            - >
+              KEYWRAP is used for securely wrapping and unwrapping encryption keys,
+              ensuring their confidentiality during transmission or storage.
+            - >
+              RADSEC is an extension of RADIUS that provides secure communication
+              between RADIUS clients and servers over TLS/SSL. Enhances enhancing the
+              confidentiality and integrity of authentication and accounting data exchange.
+            type: str
+            choices: [KEYWRAP, RADSEC]
+          message_key:
+            description:
+            - Message key used to encrypt shared secret.
+            - Updation of message key is not possible.
+            - Required when encryption_scheme is provided.
+            - >
+              When ASCII format is selected, Message Authentication Code Key may contain
+              alphanumeric and special characters. Key must be 20 char long.
+            type: str
+          encryption_key:
+            description:
+            - Encryption key used to encrypt shared secret.
+            - Updation of encryption scheme is not possible.
+            - Required when encryption_scheme is provided.
+            - >
+              When ASCII format is selected, Encryption Key may contain
```

Comment:
how do we select ASCII format? I don't see that is documented. ASCII or Hexadecimal